From 2a92b6baf452aba2cff3cc0a7782b301da3933d7 Mon Sep 17 00:00:00 2001
From: Jan Tojnar <jtojnar@gmail.com>
Date: Sun, 4 Aug 2024 05:41:28 +0200
Subject: [PATCH] strokemap: Replace deprecated tostring with tobytes

https: //numpy.org/devdocs/reference/generated/numpy.ndarray.tostring.html#numpy.ndarray.tostring
---
 lib/strokemap.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/strokemap.py b/lib/strokemap.py
index f279edf84..59b26676f 100644
--- a/lib/strokemap.py
+++ b/lib/strokemap.py
@@ -411,7 +411,7 @@
 
     """
 
-    _ZDATA_ONES = zlib.compress(np.ones((N, N), 'uint8').tostring())
+    _ZDATA_ONES = zlib.compress(np.ones((N, N), 'uint8').tobytes())
 
     def __init__(self):
         """Initialize, as a tile filled with all ones."""
@@ -451,7 +451,7 @@
             tile._zdata = None
         else:
             tile._all = False
-            tile._zdata = zlib.compress(array.tostring())
+            tile._zdata = zlib.compress(array.tobytes())
         return tile
 
     @classmethod
